src/api/WeatherComponentAPI.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import axios, { AxiosResponse } from 'axios';
+import { weatherAPIKey } from '../key.json';
+
+const WeatherComponentAPI: React.FC<unknown> = (city) => {
```

Comment:
Shall we declare the type above the component like 
type Props = {
  city: string;
};
?